src/libmongoc/tests/json/sessions/legacy/dirty-session-errors.json

```diff
@@ -21,171 +21,6 @@
     }
   ],
   "tests": [
-    {
-      "description": "Clean explicit session is not discarded",
```

Comment:
I don't know when exactly these two tests were removed from the spec, but might as well update.